src/lib/viewers/image/ImageViewer.js

```diff
@@ -17,12 +18,29 @@ class ImageViewer extends ImageBaseViewer {
         this.updatePannability = this.updatePannability.bind(this);
         this.handleImageDownloadError = this.handleImageDownloadError.bind(this);
         this.handleAssetAndRepLoad = this.handleAssetAndRepLoad.bind(this);
+        this.handleZoomEvent = this.handleZoomEvent.bind(this);
+        this.annotationControlsFSM = new AnnotationControlsFSM(
```

Comment:
`AnnotationControlsFSM` has already been created in `BaseViewer` constructor, right? Should we move the logic to `BaseViewer`? Or call something like `this.annotationControlsFSM.setState` here?

Reply:
I think @ConradJChan and I discussed that we wanted ImageViewer to instantiate its own instance of AnnotationControlsFSM.

Reply:
Let's also make sure that if `enableAnnotationsImageDiscoverability === false`, that we get the old `ImageViewer` experience, i.e. not the `enableAnnotationsDiscoverability` experience

Reply:
Could you elaborate a little on this? Do you mean that we should have a check like this:

`
if (enableAnnotationsImageDiscoverability) {
     this.annotationControlsFSM = new AnnotationControlsFSM(AnnotationState.REGION_TEMP);
}
`